src/features/lightspeed/roleGeneration.ts

```diff
@@ -140,16 +140,17 @@ export async function showRoleGenerationPage(extensionUri: vscode.Uri) {
 
   panel.onDidDispose(async () => {
     await sendActionEvent(WizardGenerationActionType.CLOSE_CANCEL, undefined);
-    currentPanel = undefined;
     wizardId = undefined;
   });
 
-  currentPanel = panel;
-  wizardId = uuidv4();
-
   panel.webview.onDidReceiveMessage(async (message) => {
     const command = message.command;
     switch (command) {
+      case "reset": {
```

Comment:
"Reset" will prompt Users to confirm [for Playbook Generation](https://issues.redhat.com/browse/AAP-34643).

I can make the same change here (after this is merged and I complete the foregoing).

Reply:
Yes, we will need a similar change.